src/ContainerDetailsSubheader.react.js

```diff
@@ -89,7 +91,7 @@ var ContainerDetailsSubheader = React.createClass({
       metrics.track('Opened In Browser', {
         from: 'header'
       });
-      exec(['open', this.state.ports[this.state.defaultPort].url], function (err) {
+      util.openPathOrUrl(this.state.ports[this.state.defaultPort].url, function (err) {
```

Comment:
Wondering if we could use: https://github.com/atom/atom-shell/blob/master/docs/api/shell.md